datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -185,6 +185,11 @@ type Mutation {
     """
     updateDataset(urn: String!, input: DatasetUpdateInput!): Dataset
 
+    """
+    Update the metadata about a batch of Datasets
+    """
+    updateDatasets(urns: [String!]!, inputs: [DatasetUpdateInput!]!): [Dataset]
```

Comment:
curious to understand what you'll be using this method for. 

Is it doing things like adding owners, adding tags, adding terms, etc? If yes, we are planning to implement batch adding metadata resolvers very soon. 

If it's about updating something else (ie description), then may this makes sense to keep!